static/js/runtime-config-listeners/analytics.js

```diff
@@ -0,0 +1,13 @@
+import { canonicalizeBoolean } from '../utils';
+
+export default {
```

Comment:
If we ever add TypeScript to the Theme, it'd be great to have a `RuntimeConfigListener` type. I could see Consulting or a more advanced client wanting to right their own listeners.

Reply:
ya, that would be great